solutions/swb-reference/integration-tests/tests/isolated/datasets/uploadFile.test.ts

```diff
@@ -7,47 +7,59 @@ import axios from 'axios';
 import { getProjectAdminRole } from '../../../../src/utils/roleUtils';
 import ClientSession from '../../../support/clientSession';
 import { DatasetHelper } from '../../../support/complex/datasetHelper';
+import { PaabHelper } from '../../../support/complex/paabHelper';
 import Setup from '../../../support/setup';
 import HttpError from '../../../support/utils/HttpError';
 import Settings from '../../../support/utils/settings';
 import { checkHttpError } from '../../../support/utils/utilities';
 
 describe('datasets file upload tests', () => {
+  const paabHelper: PaabHelper = new PaabHelper();
   const setup: Setup = new Setup();
   const settings: Settings = setup.getSettings();
-  let adminSession: ClientSession;
   let datasetHelper: DatasetHelper;
-
+  let pa1Session: ClientSession;
+  let project1Id: string;
   let dataSet: DataSet;
 
   beforeEach(async () => {
     expect.hasAssertions();
 
-    const { data } = await adminSession.resources.datasets.create({
-      storageName: settings.get('DataSetsBucketName'),
-      awsAccountId: settings.get('mainAccountId'),
-      region: settings.get('awsRegion'),
-      owner: getProjectAdminRole(settings.get('projectId')),
-      ownerType: 'GROUP',
-      type: 'internal'
-    });
+    const { data } = await pa1Session.resources.projects
+      .project(project1Id)
+      .dataSets()
+      .create({
+        storageName: settings.get('DataSetsBucketName'),
+        awsAccountId: settings.get('mainAccountId'),
+        region: settings.get('awsRegion'),
+        owner: getProjectAdminRole(project1Id),
+        ownerType: 'GROUP',
+        type: 'internal'
+      });
```

Comment:
Can probably use the `createDatasetRequest` helper from `paabHelper` here

Reply:
In the future we will refactor common code into the helper and will address this when that time comes.